### ISSUE TITLE
softmax w/ too many arguments

### ISSUE THREAD

**Issue by jdhurwitz** [rizwan09/bcn_based#1]
It appears that the torch.nn.softmax and torch.nn.functional.softmax require 1 argument (input) rather than input and dim. 

In model.py, lines 74, 75, 90, 92 all have this issue where input and dim is specified.

**Comment by jdhurwitz** [comment]
This seems to be version specific.
